libs/dataset-explorer/.eslintrc.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": ["../../.eslintrc"],
-  "ignorePatterns": ["!**/*"],
+  "ignorePatterns": ["!**/*", "LargeDatasetExplorerTab.tsx"],
```

Comment:
why do we exclude this from from linting?
Are we able to fix lint issues in the file instead of excluding checking

Reply:
The only lint problem is the number of lines is greater than 300.

Reply:
Just limiting the linting to file size now. There are no other lint issues.